api/features/release_pipelines/core/models.py

```diff
@@ -90,18 +108,14 @@ def get_delete_log_message(
     ) -> typing.Optional[str]:
         return RELEASE_PIPELINE_DELETED_MESSAGE % self.name
 
+    def has_feature_in_flight(self) -> bool:
+        return EnvironmentFeatureVersion.objects.filter(  # type: ignore[no-any-return]
```

Comment:
It pleases mypy to assign the variable with type to avoid the type ignore:
```
       has_feature_in_flight: bool = EnvironmentFeatureVersion.objects.filter(
            published_at__isnull=True, pipeline_stage__in=self.stages.all()
        ).exists()
        return has_feature_in_flight
```